halyard-deploy/src/main/java/com/netflix/spinnaker/halyard/deploy/deployment/v1/DistributedDeployer.java

```diff
@@ -318,18 +346,6 @@ public RemoteAction deploy(DistributedServiceProvider<T> serviceProvider,
     List<Integer> allOrcas = new ArrayList<>(executionsByServerGroupVersion.keySet());
     allOrcas.sort(Integer::compareTo);
 
-    int orcaCount = allOrcas.size();
-    if (orcaCount <= MAX_REMAINING_SERVER_GROUPS) {
-      return;
-    }
-
-    allOrcas = allOrcas.subList(0, orcaCount - MAX_REMAINING_SERVER_GROUPS);
-    for (Integer orcaVersion : allOrcas) {
-      // TODO(lwander) consult clouddriver to ensure this orca isn't enabled
```

Comment:
TODO was dropped because the orca versions outside of MAX_REMAINING_SERVER_GROUPS cannot be enabled, since the rollback operation requested by halyard ensures the highest sequence number is always the only one serving traffic.